app/webpacker/components/Results/resultsFilter.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from 'react';
+import {
+  Button, ButtonGroup, Form, Segment,
+} from 'semantic-ui-react';
+import { EventSelector } from '../wca/EventSelector';
+import { RegionSelector } from '../CompetitionsOverview/CompetitionsFilters';
+import { countries } from '../../lib/wca-data.js.erb';
+import I18n from '../../lib/i18n';
+
+export default function ResultsFilter({ filterState }) {
+  const {
+    event,
+    setEvent,
+    region,
+    setRegion,
+    rankingType,
+    setRankingType,
+    gender,
+    setGender,
+    show,
+    setShow,
+  } = filterState;
+  const regionIso2 = useMemo(() => {
+    if (region === 'world') {
+      return 'all';
+    }
+    const iso2 = countries.real.find((country) => country.id === region)?.iso2;
+    if (iso2) {
+      return iso2;
+    }
+    return region;
+  }, [region]);
+  return (
+    <Segment raised>
+      <Form>
+        <Form.Field>
+          <EventSelector
```

Comment:
Maybe we should add a new prop `title` with default value `"Events"`, and use `title="Event"` here, since you can only select one.

If so, it would make sense to use the title `"Event"` in the psych sheet too.

Alternatively, we could automatically make the title `"Event"` if the 'All' button is hidden?